UnityProject/Assets/Scripts/Managers/PlayerList.Admin.cs

```diff
@@ -433,10 +435,16 @@ private async Task<bool> CheckUserState(string unverifiedToken, ConnectedPlayer
 		//Must have non-null/empty username
 		if (string.IsNullOrEmpty(unverifiedConnPlayer.Username))
 		{
-			StartCoroutine(KickPlayer(unverifiedConnPlayer, $"Server Error: Account has invalid username (Null/Empty). To fix go to character creator then click Serialise and then load"));
-			Logger.LogError($"A user tried to connect with null/empty username" +
-			           $"Details: Username: {unverifiedConnPlayer.Username}, ClientID: {unverifiedConnPlayer.ClientId}, IP: {unverifiedConnPlayer.ConnectionIP}",
-				Category.Admin);
+			var prefsCheck = PlayerPrefs.GetString("currentcharacter");
+			if (string.IsNullOrEmpty(prefsCheck))
+			{
+				StartCoroutine(KickPlayer(unverifiedConnPlayer, $"Server Error: Account has invalid username (Null/Empty). To fix go to character creator then click Serialise and then load"));
+				Logger.LogError($"A user tried to connect with null/empty username" +
+				                $"Details: Username: {unverifiedConnPlayer.Username}, ClientID: {unverifiedConnPlayer.ClientId}, IP: {unverifiedConnPlayer.ConnectionIP}",
+					Category.Admin);
+			}
+			LobbyManager.Instance.characterCustomization.SetActive(true);
```

Comment:
isn't this only server side?

Reply:
maybe? should I turn that into a net message?

Reply:
or rpc

Reply:
Done 👍